### PR TITLE
Update replicated local storage to use OpenEBS instead of OpenEBS Jiva which is now deprecated

### LINKED ISSUE #9062: Request updating "Storage" / "Local Storage" / "Replicated Local Storage" content in "Documentation"
- OpenEBS Jiva is now deprecated
- new OpenEBS Storage contains both Local Storage ( for talos, it is Local PV Hostpath) and Replicated Storage ( Mayastor only )
- Prep of OpenEBS Storage like below
```yaml
    machine:
      sysctls:
        vm.nr_hugepages: "1024"
      nodeLabels:
        openebs.io/engine: mayastor
      kubelet:
        extraMounts:
          - destination: /var/local/openebs
            type: bind
            source: /var/local/openebs
            options:
              - rbind
              - rshared
              - rw
```
- Installation of OpenEBS storage is simplified, like below
```shell
helm repo add openebs https://openebs.github.io/openebs
helm repo update

helm install openebs \
     --create-namespace \
     --namespace openebs \
     --set engines.local.lvm.enabled=false \
     --set engines.local.zfs.enabled=false \
     --set mayastor.etcd.clusterDomain="cluster.domain.here" \
     openebs/openebs
```
- other details about how to use new OpenEBS Storage, please check new [doc](https://openebs.io/docs/)

### PR DESCRIPTION
**Pull request by v1nsai**
# Pull Request

## What? (description)
As per [this announcement](https://github.com/openebs/openebs/issues/3709), OpenEBS has deprecated `openebs-jiva`.  This PR updates the local-replicated-storage instructions in the documentation to use the standard `openebs`.  This PR resolves #9062 

## Why? (reasoning)
The official documentation should encourage using the new project rather than the deprecated one.
## Acceptance

Please use the following checklist:

- [x] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [ ] you ran conformance (`make conformance`)
- [ ] you formatted your code (`make fmt`)
- [ ] you linted your code (`make lint`)
- [ ] you generated documentation (`make docs`)
- [ ] you ran unit-tests (`make unit-tests`)

> See `make help` for a description of the available targets.

### CODE REVIEW

**Review by smira — APPROVED** [review]
thank you!

File: website/content/v1.8/kubernetes-guides/configuration/replicated-local-storage-with-openebs.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+```yaml
+machine:
+  sysctls:
+    vm.nr_hugepages: "1024"
+  nodeLabels:
+    openebs.io/engine: mayastor
+  kubelet:
+    extraMounts:
+      - destination: /var/local/openebs
```
Comment (fullmetal-fred) [inline website/content/v1.8/kubernetes-guides/configuration/replicated-local-storage-with-openebs.md:31]: This path is incorrect based on the standard helm chart...should be `/var/openebs/local`. Change must be made below as well.

File: website/content/v1.8/kubernetes-guides/configuration/replicated-local-storage-with-openebs.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        type: bind
+        source: /var/local/openebs
```
Comment (fullmetal-fred) [inline website/content/v1.8/kubernetes-guides/configuration/replicated-local-storage-with-openebs.md:33]: See above.

**Review by fullmetal-fred — COMMENTED** [review]
The path in the extraMounts config is incorrect based on openebs defaults.

File: website/content/v1.8/kubernetes-guides/configuration/replicated-local-storage-with-openebs.md (hunk 1 above)
Comment (v1nsai) [reply website/content/v1.8/kubernetes-guides/configuration/replicated-local-storage-with-openebs.md:31]: can we reopen or should I make another PR?

Comment (smira) [reply website/content/v1.8/kubernetes-guides/configuration/replicated-local-storage-with-openebs.md:31]: This PR is merged, so please open another PR

Comment (v1nsai) [reply website/content/v1.8/kubernetes-guides/configuration/replicated-local-storage-with-openebs.md:31]: Done https://github.com/siderolabs/talos/pull/9155

**Pull request merged**